discovery/api/v1/api.go

```diff
@@ -55,6 +58,12 @@ func (w *Wrapper) Routes(router core.EchoRouter) {
 				ctx.Set(core.OperationIDContextKey, operationID)
 				ctx.Set(core.ModuleNameContextKey, discovery.ModuleName)
 				ctx.Set(core.StatusCodeResolverContextKey, w)
+				// deepmap/openapi codegen does not support dynamic query parameters ("exploded form parameters"),
```

Comment:
is this a feature of the strict-server option?

Reply:
Not sure, but it's a rather obscure/new feature which not all code generators support properly